petastorm/reader_impl/reader_v2.py

```diff
@@ -156,11 +155,13 @@ def __init__(self, dataset_url, schema_fields=None, shuffle=None, predicate=None
         epoch_items = self._apply_row_drop_partition(filtered_row_groups, shuffle_options)
 
         # 4. Launch a new thread running `worker_loop` function.
-        def epochs_iterator(): return epoch_generator(epoch_items, num_epochs, shuffle_options.shuffle_row_groups)
+        def epochs_iterator():
+            return epoch_generator(epoch_items, num_epochs, shuffle_options.shuffle_row_groups)
 
-        self._results_queue = Queue(_OUTPUT_QUEUE_SIZE)
+        self._results_queue = Queue()
```

Comment:
This removed the size of the results queue. It looks like it can now grow indefinitely?

Reply:
The responsibility of the output queue control size is now worker_loop's. The code was missing, resurrected it.